recipes/pysteps/meta.yaml

```diff
@@ -0,0 +1,69 @@
+{% set name = "pysteps" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: a2443ed3cd1636e60622a320cf7e7e8a0e857ad456e034ef19d084545ad1bc0e
+
+build:
+  number: 0
+  skip: True  # [py<36]"
+  script: "{{ PYTHON }} -m pip install . --no-deps -vv"
+
+requirements:
+  build:
+    - numpy
+    - cython
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - numpy
+    - cython
+    - pip
+    - openmp 4.0.*  # [osx]
+    - attrdict
+    - jsmin
+    - scipy
+    - matplotlib
```

Comment:
The `host` section should only contain dependencies which are needed to run `setup.py`.

Are these all needed or can some of them be removed? (`jsonschema`, `matplotlib`, `scipy` and probably some of the others)

Reply:
Those dependencies are passed to setup() in setup.py, trough the *requirements* keyword. If we  don't include them conda-build gives the following error: 

`RuntimeError: Setuptools downloading is disabled in conda build. Be sure to add all dependencies in the meta.yaml`